index.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const NR = require('node-resque');
+const asCallback = require('ascallback');
+const config = require('config');
+const winston = require('winston');
```

Comment:
When running under scale winston is not performant, I suggest pino/debug (we could change later as well)

https://github.com/pinojs/pino#benchmarks
https://medium.com/the-node-js-collection/get-ready-a-new-v8-is-coming-node-js-performance-is-changing-46a63d6da4de (See the last section on logger comparison on various v8 engines)